promql/parser/lex.go

```diff
@@ -429,11 +429,10 @@ func lexStatements(l *Lexer) stateFn {
 			l.emit(EQL)
 		}
 	case r == '!':
-		if t := l.next(); t == '=' {
-			l.emit(NEQ)
-		} else {
+		if t := l.next(); t != '=' {
```

Comment:
Newest golangci-lint requires this change.

Reply:
I love this lint change =D